
        /// &lt;summary&gt;
        /// ステータスコードが69
        /// &lt;/summary&gt;
        [TestCaseSource(nameof(MyTestDataName))]
        public void UpdateError(TestModel inputModel)
        {
            var function = new TestFunction&lt;MyDBContextName&gt;(_context);
            // テストデータをInsert
            var list = GetTestData();
            _context.PrPrint.AddRange(GetTestData());
            _context.SaveChanges();
            var ex = Assert.Throws&lt;ApplicationException&gt;(async() =&gt;
                await function.Update_Test(inputModel));
            Assert.IsInstanceOf&lt;ApplicationException&gt;(ex);
            Assert.That("状態69であるデータの更新はできません", Is.EqualTo(ex.Message));
        }

